src/lib/theme.ts

```diff
@@ -32,4 +32,5 @@ export default {
   ],
   lineHeights: [2, 1.5, 1.2],
   maxWidths: [["66.66vw", "55vw"], ["100vw", "90vw"]],
+  breakpoints: [0, 600, 1600],
```

Comment:
Where is this used?

Reply:
It is automatically used by onno. If you write a style like this `maxWidth: [0, 2, 1]` each value corresponds to each of those breakpoints.

Reply:
We need some documentation 😬

Reply:
Yeah I definitely need to write some out